README.rst

```diff
@@ -128,6 +128,24 @@ openwisp-ipam provides a graphical representation of a subnet which shows the av
 REST API
 ********
 
+Live documentation
+==================
+
+.. image:: https://github.com/openwisp/openwisp-ipam/raw/master/docs/images/api-docs.png
+
+A general live API documentation (following the OpenAPI specification) is available at ``/api/v1/docs/``.
+
+Browsable web interface
+=======================
+
+.. image:: https://github.com/openwisp/openwisp-ipam/raw/master/docs/images/api-ui.png
```

Comment:
I think we should show something more interesting than "Authentication credentials were not provided".

I suggest to log in the admin, try creating a subnet, then open this page again and take a screenshot of the list and if you can include part of the POST form that would be great.

Reply:
I have added the POST form image.